src/pages/groupComparison/UpSet.tsx

```diff
@@ -60,9 +60,9 @@ const MAX_LABEL_WIDTH = 200;
 
 const BarComponent = (props: any) => (
     <Bar
-        className={`${props.caseType}_${_.sortBy(props.datum.groups).join(
-            '_'
-        )}_bar`}
+        className={`${props.caseType}_${_.sortBy(props.datum.groups)
+            .map(g => g.replace(/ /g, '_'))
```

Comment:
curious why you need to make this class

Reply:
for selecting during e2e test. im not sure why I didn't do data-test. is there a problem with using class?